onnxruntime/python/tools/kernel_explorer/kernel_explorer.cc

```diff
@@ -4,10 +4,10 @@
 #include <pybind11/pybind11.h>
 #include <pybind11/numpy.h>
 #include "python/tools/kernel_explorer/device_array.h"
-#include "python/tools/kernel_explorer/kernels/vector_add.h"
-#include "python/tools/kernel_explorer/kernels/fast_gelu.h"
 #include "python/tools/kernel_explorer/kernels/gemm.h"
-#include "python/tools/kernel_explorer/kernels/skip_layer_norm.h"
+#include "python/tools/kernel_explorer/kernels/vector_add.h"
```

Comment:
Why are the "vector_add" and "gemm" kernels not rocm specific?

Reply:
Add first I was try to keep the registration code shared. But it seems that we better make the registration distributed, instread of centralized, so I also move gemm into `rocm` dir. But vector add is a shared, so it still live out side of cuda and rocm.